samcli/commands/build/command.py

```diff
@@ -149,6 +172,8 @@ def cli(
     parallel: bool,
     manifest: Optional[str],
     docker_network: Optional[str],
+    container_env_vars: list,
+    container_env_vars_file: str,
```

Comment:
I am not sure about this one, if `required=False` for both of them in click, should we add `Optional` to them?

```suggestion
    container_env_vars: Optional[List[str]],
    container_env_vars_file: Optional[str],
```

Reply:
addressed